openvdb/tools/PointPartitioner.h

```diff
@@ -465,13 +409,13 @@ template<typename PointIndexType>
 struct MergeBinsOp
 {
     using Segment = Array<PointIndexType>;
-    using SegmentPtr = typename Segment::Ptr;
+    using SegmentPtr = std::unique_ptr<Array<PointIndexType>>;
```

Comment:
Is there a need for this change seeing as you've changed the `Ptr` typedef? We use `Ptr` and `UniquePtr`, though the `UniquePtr` typedef is something we've only used in /points/. We should decide on how we want these to be promoted, either all through `Ptr` or introducing specific types for unique, weak etc

Reply:
> Is there a need for this change seeing as you've changed the Ptr typedef?

Not really, I can change this back if you would prefer? This is only exposed in an "internal" namespace, so I'm not sure it matters too much in this specific case, though I agree there's a more general discussion to be had here (which I'd rather not get into here).

Reply:
Fixed.